docs/_Sidebar.md

```diff
@@ -18,9 +17,9 @@
   - [Schemas](schemas.md)
 
 # Knowledge content
-  - [How to run the plugin](how-to-run.md)
-  - [Example test plan](example-test-plan.md)
+  - [How to run a test plan](how-to-run.md)
+  - [Test plan example](example-test-plan.md)
  
-# Additional information
-  - [Overview](additional-information.md)
+# [Additional information](additional-information.md)
+  - [Contributing](contributing.md)
```

Comment:
I have moved this page with contributions guidelines to the corresponding module. Previously, it was at Getting Started. Please, let me know if that was intentional to gain visibility to the contributions instructions.